tests/chains/Ontology/OntTests.cpp

```diff
@@ -37,25 +35,37 @@ TEST(OntologyOnt, queryBalance) {
 }
 
 TEST(OntologyOnt, transfer) {
-    auto signer1 = Signer(
-        PrivateKey(parse_hex("4646464646464646464646464646464646464646464646464646464646464646")));
-    auto signer2 = Signer(
-        PrivateKey(parse_hex("4646464646464646464646464646464646464646464646464646464646464652")));
+    PrivateKey privateKey1(parse_hex("4646464646464646464646464646464646464646464646464646464646464646"));
+    Signer signer1(privateKey1);
+
+    PrivateKey privateKey2(parse_hex("4646464646464646464646464646464646464646464646464646464646464652"));
+    Signer signer2(privateKey2);
+
     auto toAddress = Address("Af1n2cZHhMZumNqKgw9sfCNoTWu9de4NDn");
     uint32_t nonce = 0;
     uint64_t amount = 1, gasPrice = 500, gasLimit = 20000;
     auto tx = Ont().transfer(signer1, toAddress, amount, signer2, gasPrice, gasLimit, nonce);
-    auto rawTx = hex(tx.serialize());
+    auto rawTx = tx.serialize();
+    auto rawTxHex = hex(rawTx);
+
     EXPECT_EQ("00d100000000f401000000000000204e00000000000057e9d1a61f9aafa798b6c7fbeae35639681d7df6"
               "7100c66b14fbacc8214765d457c8e3f2b5a1d3c4981a2e9d2a6a7cc814feec06b79ed299ea06fcb94aba"
               "c41aaf3ead76586a7cc8516a7cc86c51c1087472616e7366657214000000000000000000000000000000"
               "00000000010068164f6e746f6c6f67792e4e61746976652e496e766f6b65000241407531e7d5bb9ae138"
-              "862585a65c26d624f1a7a61011298809d9ed9cf60d10a4504067dee9d549a836b480c4e48904e28f9b42"
-              "dd5fa14376cbb1ef27d931eaea552321031bec1250aa8f78275f99a6663688f31085848d0ed92f1203e4"
+              "862585a65c26d624f1a7a61011298809d9ed9cf60d10a450bf9821152ab657ca4b7f3b1b76fb1d7021a4"
```

Comment:
Where did you get this data from? Respectively, is it okay to just change this?

Reply:
This shouldn't cause problems since the signatures are valid (verified by using an associated public key), and blockchain validators should accept transactions with such signature.
> Can you explain this some more? Won't this cause problems?

I tested where the transaction signature starts and replaced it with the new one.
To validate if the changed signature is valid, I've added the signature verifying with the legacy trezor-crypto verifying function to every test where I changed the signature.
> Where did you get this data from?

https://github.com/trustwallet/wallet-core/blob/d3787137647636e4374ffd14b19557402d03afcc/tests/chains/Ontology/TWAnySignerTests.cpp#L94-L100

To be honest, I'm not sure. We should test the Ontology and NEO transactions carefully.
> Respectively, is it okay to just change this?